### DISCUSSION TITLE
Add Monero (XMR) support for payments of SimpleLogin Premium

### DISCUSSION THREAD

**Discussion by sethforprivacy** [simple-login/app#511] [Feature Requests]
Monero is a decentralized and open-source cryptocurrency that protects both sender and receiver privacy by default for all users. It has a vibrant pro-privacy community and meshes well with a service like Simple Login. More information about Monero can be found here:

https://www.getmonero.org/get-started/what-is-monero/

Would it be possible to get Monero added as a supported cryptocurrency? There are lots of good details available here around the benefits for you and your customers, as well as how to integrate Monero:

https://www.monerooutreach.org/merchants/

A few specific options for Monero acceptance are:

- https://btcpayserver.org/ (my recommendation)
  - Self-hosted and entirely non-custodial/involves no 3rd party payment processors while supporting a wide variety of cryptocurrencies (including Monero)
- https://globee.com/
- https://www.coinpayments.net/

If you do decide to integrate Monero payments and donations, please don't hesitate to share in https://www.reddit.com/r/Monero/ and on Twitter, tagging the @monero account or myself @sethforprivacy.

Please don't hesitate to reach back out to me if you have more questions about the integration or Monero in general.

**Comment by nanostos** [comment]
Excellent idea. Simple Login is a privacy-preserving tool and Monero is the gold standard for privacy-preserving payments. It should really be the default payment method.

I would add [https://nowpayments.io/supported-coins/monero-payments](https://nowpayments.io/supported-coins/monero-payments) to the list of merchant tools, as they seem to be quick and easy to implement.

**Comment by Yolakalemowa** [comment]
what happened to this request? I still don't see them accepting any crypto payments, not to mention XMR.

**Reply by nguyenkims** [reply]
SL already supports XMR via our partner https://digitalgoods.proxysto.re/en
